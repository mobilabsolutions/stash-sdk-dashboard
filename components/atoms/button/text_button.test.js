/* eslint-env jest */
import React from 'react'
import renderer from 'react-test-renderer'

import { withTestSetup } from '../../../test_utils'
import TextButton from './text_button'

const TestTextButton = withTestSetup(TextButton)

it('TextButton Should Render', () => {
  const component = renderer.create(
    <TestTextButton>Sample Text</TestTextButton>
  )
  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
