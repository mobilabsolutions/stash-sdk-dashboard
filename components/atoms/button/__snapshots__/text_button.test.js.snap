// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TextButton Should Render 1`] = `
<button
  className="text_button-grth4y-0 cCjzYt"
>
  Sample Text
</button>
`;
